unstable/wlr-screencopy-unstable-v1.xml

```diff
@@ -160,6 +160,14 @@
            summary="low 32 bits of the seconds part of the timestamp"/>
       <arg name="tv_nsec" type="uint"
            summary="nanoseconds part of the timestamp"/>
+      <arg name="damage_x" type="uint"
+           summary="current x coordinates for damage"/>
+      <arg name="damage_y" type="uint"
+           summary="current y coordinates for damage"/>
+      <arg name="damage_width" type="uint"
+           summary="current width of damage"/>
+      <arg name="damage_height" type="uint"
+           summary="current height of damage"/>
```

Comment:
Adding arguments to an existing event is not backwards-compatible. Add a new damage event instead.

Reply:
I forgot to remove it when I added the damage event. It is removed now.